fastlane/lib/fastlane.rb

```diff
@@ -19,6 +19,7 @@
 require 'fastlane/documentation/docs_generator'
 require 'fastlane/other_action'
 require 'fastlane/plugin_manager'
+require 'fastlane/plugins/plugins'
```

Comment:
Yeah, now that we have a group of classes, I think it might make sense to move the `plugin_manager` in there too.